src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-items: center; /* Вирівняти по центру */
+  justify-content: space-between;
+  margin-left: 16px;
+  margin-right: 16px;
+
+  &__active {
+    display: flex;
+  }
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(5) {
+    background-image: url(../images/star.svg);
```

Comment:
The active background image should depend on the modifier class. Consider using CSS from the previous task to keep modifiers working

Reply:
done